test/dotnet-msbuild.Tests/GivenDotnetCleanInvocation.cs

```diff
@@ -10,14 +10,14 @@ namespace Microsoft.DotNet.Cli.MSBuild.Tests
 {
     public class GivenDotnetCleanInvocation
     {
-        const string ExpectedPrefix = "exec <msbuildpath> /m /v:m /v:normal /t:Clean";
+        const string ExpectedPrefix = "exec <msbuildpath> -m -v:m /v:normal /t:Clean";
 
         [Fact]
         public void ItAddsProjectToMsbuildInvocation()
         {
             var msbuildPath = "<msbuildpath>";
             CleanCommand.FromArgs(new string[] { "<project>" }, msbuildPath)
-                .GetProcessStartInfo().Arguments.Should().Be("exec <msbuildpath> /m /v:m /v:normal <project> /t:Clean");
+                .GetProcessStartInfo().Arguments.Should().Be("exec <msbuildpath> -m -v:m /v:normal <project> /t:Clean");
```

Comment:
I'm not loving the mix of - and /. I think we should define 1 style and use it *everywhere*. I would favor long form and - for *everything*.

Reply:
When I made the change I expected the same thing. This seems mechanical and simple. I will get to it. Will change all our implicit transformations to - instead of / and will move to long form.

Reply:
👍 

Don't forget about this place where we assume we're using /p:TargetFramework= elsewhere: https://github.com/dotnet/cli/blob/6b240926a3f4a71ac097ba234bd4f627efaedcad/src/dotnet/commands/RestoringCommand.cs#L733

Reply:
I guess we should search whole code base for all of the msbuild switches.